src/component/MembersList.jsx

```diff
@@ -27,7 +48,15 @@ const MembersList = ({teamId}) => {
   return (
     <div className="containerList">
       <div className="titleContainer">
-        <h4>Here is a list of your team members!</h4>
+        <section>

```

Comment:
Shouldn’t the condition also include the `<section>` element itself? If the team does not exist, the `<section>` element will still be added to the DOM without any value, which might not be ideal.